src/codegen/llvm.zig

```diff
@@ -5585,6 +5586,7 @@ pub const FuncGen = struct {
         switch (modifier) {
             .auto, .never_tail, .always_tail => {},
             .never_inline => try attributes.addFnAttr(.@"noinline", &o.builder),
+            .never_intrinsify => try attributes.addFnAttr(.nobuiltin, &o.builder),
```

Comment:
Note that the handling of `never_intrinsify` will need to be combined with the code I added here: https://github.com/ziglang/zig/pull/21920/files#diff-4b7d5a9d78d51e4f41a41d80317e1ca360c16103d76b09c3dcb73ab3e4cc2d71R5579-R5581